spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -265,4 +265,126 @@ def over_65_birth_year
+      it 'returns nil' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 10
+        allow(instance).to receive(:calculate_line_42).and_return 80_001
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(nil)
+      end
+    end
+
+    context 'when taxable income is under 80k and taxpayer has 1 dependent' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
+      it 'returns 1000 for incomes less than or equal to 30k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
+        allow(instance).to receive(:calculate_line_42).and_return 30_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(1000)
+      end
+
+      it 'returns 800 for incomes less than or equal to 40k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
+        allow(instance).to receive(:calculate_line_42).and_return 40_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(800)
+      end
+
+      it 'returns 600 for incomes less than or equal to 50k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
+        allow(instance).to receive(:calculate_line_42).and_return 50_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(600)
+      end
+
+      it 'returns 400 for incomes less than or equal to 60k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
+        allow(instance).to receive(:calculate_line_42).and_return 60_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(400)
+      end
+
+      it 'returns 200 for incomes less than or equal to 80k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
+        allow(instance).to receive(:calculate_line_42).and_return 80_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(200)
+      end
+    end
+
+    context 'when all dependents are over 5 years old' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
+
+      it 'returns nil' do
+        five_years_one_day = Date.new(MultiTenantService.new(:statefile).current_tax_year - 6, 12, 31)
+        intake.synchronize_df_dependents_to_database
+        intake.dependents.first.update(dob: five_years_one_day)
+        intake.dependents.reload
+        allow(instance).to receive(:calculate_line_42).and_return 10_000
+        instance.calculate
+        expect(intake.dependents.count).to eq(1)
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(nil)
+      end
+    end
+
+    context 'with multiple dependents, some of whom are under 5 years old' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_many_deps) }
+
```

Comment:
[pebble] same here could add for example and separate the ones with different setups by context
```
before do
 five_years = Date.new(MultiTenantService.new(:statefile).current_tax_year - 5, 1, 1)
        five_years_one_day = Date.new(MultiTenantService.new(:statefile).current_tax_year - 6, 12, 31)
        intake.synchronize_df_dependents_to_database
        intake.dependents.each do |d| d.update(dob: five_years_one_day) end
        intake.dependents.first.update(dob: five_years)
        intake.dependents.reload
        allow(instance).to receive(:calculate_line_42).and_return 10_000
        instance.calculate
end
```